predict_pv_yield/models/loss.py

```diff
@@ -0,0 +1,37 @@
+import math
+import torch
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class WeightedLosses:
+    def __init__(self, decay_rate: int = None, forecast_length: int = 6):
```

Comment:
Given that `decay_rate` can be `None`, I'd recommend using this type annotation:  

```python
decay_rate: Optional[int] = None
```

Just to make it super-explicit that `decay_rate` can be `None`!  :)

Reply:
Yea, I like it